bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -0,0 +1,234 @@
+                    self.player_dict[ctx.channel.id][msg.author] = points
+                hint_no = 0
+                unanswered = 0
+                await ctx.send(f"{msg.author.mention} got the correct answer :tada: {points} points for ya.")
+                await self.send_answer(ctx.channel, question_dict)
+                await self.send_score(ctx.channel, self.player_dict[ctx.channel.id])
+                await asyncio.sleep(2)
+
+    @staticmethod
+    async def send_score(channel: discord.TextChannel, player_data: dict) -> None:
+        """A function which sends the score."""
+        embed = discord.Embed(colour=discord.Colour.blue())
+        embed.title = "Score Board"
+        embed.description = ""
+        for k, v in player_data.items():
+            embed.description += f"{k} : {v}\n"
+        await channel.send(embed=embed)
+
+    @staticmethod
+    async def declare_winner(channel: discord.TextChannel, player_data: dict) -> None:
+        """Announce the winner of the quiz in the game channel."""
+        if player_data:
+            highest_points = max(list(player_data.values()))
+            no_of_winners = list(player_data.values()).count(highest_points)
+
+            # Check if more than 1 player has highest points.
+            if no_of_winners > 1:
+                word = "You guys"
+                winners = []
+                points_copy = list(player_data.values()).copy()
+                for _ in range(no_of_winners):
+                    index = points_copy.index(highest_points)
+                    winners.append(list(player_data.keys())[index])
+                    points_copy[index] = 0
+                winners_mention = None
+                for winner in winners:
+                    winners_mention += f"{winner.mention} "
+
+            else:
+                word = "You"
+                author_index = list(player_data.values()).index(highest_points)
+                winner = list(player_data.keys())[author_index]
+                winners_mention = winner.mention
+            await channel.send(
+                f"Congratz {winners_mention} :tada: "
+                f"{word} have won this quiz game with a grand total of {highest_points} points!!"
+            )
+
+    @property
+    def category_embed(self) -> discord.Embed:
+        """Build an embed showing all available trivia categories."""
+        embed = discord.Embed(colour=discord.Colour.blue())
+        embed.title = "The available question categories are:"
+        embed.description = ""
+        for cat, description in self.categories.items():
+            embed.description += f"**- {cat.capitalize()}**\n{description.capitalize()}\n"
+        embed.set_footer(text="If not category is chosen, then a random one will be selected.")
+        return embed
+
+    @staticmethod
+    async def send_answer(channel: discord.TextChannel, question_dict: dict) -> None:
+        """Send the correct the answer of a question to the game channel."""
```

Comment:
```suggestion
        """Send the correct answer of a question to the game channel."""
```
My mistake, I was pretty worn out by the time I gave the suggestion for this docstring.